internal/handler/handlerseller.go

```diff
@@ -0,0 +1,85 @@
+package handler
+
+import (
+	"net/http"
+	"strconv"
+	"github.com/bootcamp-go/web/response"
+	"github.com/go-chi/chi/v5"
+	"github.com/maxwelbm/alkemy-g7.git/internal/service"
+)
+
+type SellersJSON struct {
+	ID          int    `json:"id"`
+	CID         int    `json:"cid"`
+	CompanyName string `json:"company_name"`
+	Address     string `json:"address"`
+	Telephone   string `json:"telephone"`
+}
+
+func CreateHandlerSellers(service service.SellersService) *SellersController {
+	return &SellersController{service: service}
+}
+
+type SellersController struct {
+	service service.SellersService
+}
+
+func (hd *SellersController) GetAllSellers() http.HandlerFunc {
```

Comment:
Podemos simplificar essa abordagem e só ultilizamos quando realmente for necessario
```suggestion
func (hd *SellersController) GetAllSellers(w http.ResponseWriter, r *http.Request)  {
```